src/trello_conn.py

```diff
@@ -0,0 +1,56 @@
+from trello import TrelloClient
+import requests
+
+
+def card_already_in_trello(card_id):
+    with open("trello_cards") as file:
+        lines = file.readlines()
+    if f"{card_id}\n" in lines:
+        return True
+    else:
+        with open('trello_cards', 'a') as file:
+            file.write(f'{card_id}\n')
+        return False
+
+
+class TrelloConn:
+    def __init__(self, api_key, token, board_id):
+        self.api_key = api_key
+        self.token = token
+        self.board_id = board_id
+
+    def add_new_card(self, formatter_obj, list_name=None, client=None):
+        info = formatter_obj.name_and_id()
+        card_name = info[0]
+        card_id = info[1]
+        if card_already_in_trello(card_id):
+            return
+        card_description = formatter_obj.format_markdown()
+        if client is None:
+            client = TrelloClient(self.api_key, self.token)
+        board = client.get_board(self.board_id)
+        lists = board.all_lists()
+        selected_list = None
+        for trello_list in lists:
+            # if None return first, unless it is archived
+            if (list_name is None and not trello_list.closed) or trello_list.name == list_name:
+                selected_list = trello_list
+                break
+        if selected_list is not None:
+            selected_list.add_card(card_name, card_description)
+        else:
+            print(f"No list found with a given name: {list_name}")
```

Comment:
Use `logging` from Python

Reply:
From what I can see, that would be the only use of ```logging``` from python in this project. Do you really want me to add a log file or for example throw an exception here?

Reply:
Not the [first](https://github.com/RouNNdeL/aiesec-reception-tool/blob/e3e079881e6e577c3bc7e052f33aeb3b5d88e22e/src/main.py#L51), certainly not the last

Reply:
But the exception fits better here, you are right